scanners/wpscan/Chart.yaml

```diff
@@ -4,8 +4,8 @@ description: A Helm chart for the WordPress security scanner that integrates wit
 
 type: application
 # version - gets automatically set to the secureCodeBox release version when the helm charts gets published
-version: v2.5.0-alpha1
-appVersion: latest
+version: v2.6.0-alpha1
+appVersion: v3.8.15
```

Comment:
We could pin the image to the current digest, otherwise the 3.8.15 will be wrong after the next wpscan update.

Reply:
Mhh yeah, but the problem is that there is only a latest tag available an no versioning schema https://hub.docker.com/r/wpscanteam/wpscan/tags?page=1&ordering=last_updated 👎 

I guess pinning to a specific digest wouldn't solve this problem :-/

I'll revert this change here back to latest and hopefully the wpscan will fix that in the feature an support tags with release versions.